portality/view/api_v1.py

```diff
@@ -55,7 +55,7 @@ def docs():
 @swag(swag_summary='Search your applications <span class="red">[Authenticated, not public]</span>', swag_spec=DiscoveryApi.get_application_swag())  # must be applied after @api_key_(optional|required) decorators. They don't preserve func attributes.
 @blueprint.route("/search/applications/<path:search_query>")
 @api_key_required
-@track_analytics('API Hit', 'Search applications', record_value_of_which_arg='search_query')
+@analytics.sends_ga_event('API Hit', 'Search applications', record_value_of_which_arg='search_query')
```

Comment:
I'm quite keen to get rid of "magic strings" in our code, so it would be good to place "API Hit" and "Search applications" into a constants file that we just reference.  I've done this in the Update Requests code, too, if you want to see an example of one approach.  For the moment, just having these events listed somewhere and referenced would be great.  That way at least we guarantee consistent spelling and capitalisation, with IDE-level checking that we're using the right values, and runtime guarantee that the values are set.